src/components/BookFormRedux.js

```diff
@@ -1,56 +1,59 @@
 /* eslint-disable */
 import React from "react";
+import { useState } from "react";
 import { useDispatch } from "react-redux";
-import { addBook } from '../redux/books/booksSlice'
+import { addBook, addBookAsync } from "../redux/books/booksSlice";
 
 
 const BookForm = () => {
-    const [book, setBook] = React.useState({
-        'title': '',
-        'author': '',  
-    })
 
+    const [title, setTitle] = useState('');
+    const [author, setAuthor] = useState('');
     const dispatch = useDispatch();
+    
 
-    const handleChange = (e) => {
-        setBook((book) => ({
-            ...book,
-            [e.target.name]: e.target.value
-          }));
-    };
+    // const handleChange = (e) => {
+    //     setBook((book) => ({
+    //         ...book,
+    //         [e.target.name]: e.target.value
+    //       }));
+    // };
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: